src/main/java/eu/europeana/downloads/GetRecordQuery.java

```diff
@@ -71,7 +65,7 @@ private void execute(OAIPMHServiceClient oaipmhServer, String currentIdentifier)
                 ZipUtility.writeInZip(zout, writer, record);
 
             }
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
It's best to not catch Exception as this also includes things like OutOfMemory Better use something like catch one or more specifc exception or if you really want to be more generic catch RuntimeException e (see also http://sonarqube.eanadev.org:9000/coding_rules?open=squid%3AS2221&rule_key=squid%3AS2221)

Reply:
I don't think we need to catch anything there. Previously there was a ZipOutputStream and OutputStreamWriter in the try block. Now as I have removed that. We can remove this try-catch block as well